### PR TITLE
Update router terminology

### PR DESCRIPTION
**Pull request by Meschreiber**
Updates uses of "Apollo Router" to "GraphOS Router" or "Apollo Router Core" where necessary

### CODE REVIEW

File: docs/shared/enterprise-directive.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <EnterpriseFeature>
 
-This directive is an [Enterprise feature](/router/enterprise-features) of the Apollo Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+This directive is an [Enterprise feature](/router/enterprise-features) of the Apollo Router Core and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```
Comment (shorgi) [inline docs/shared/enterprise-directive.mdx:3]: Enterprise feature = GraphOS Router
```suggestion
This directive is an [Enterprise feature](/router/enterprise-features) of the GraphOS Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```

File: docs/shared/progressive-override-enterprise.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <EnterpriseFeature>
 
-Progressive `@override` is an [Enterprise feature](/router/enterprise-features) of the Apollo Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+Progressive `@override` is an [Enterprise feature](/router/enterprise-features) of the Apollo Router Core and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```
Comment (shorgi) [inline docs/shared/progressive-override-enterprise.mdx:3]:
```suggestion
Progressive `@override` is an [Enterprise feature](/router/enterprise-features) of the GraphOS Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). If your organization doesn't have an Enterprise plan, you can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```

File: docs/source/quickstart/setup.mdx [hunk 3]
```diff
@@ -194,17 +194,17 @@ Complete the first two steps of <a href="/rover/configuring/" target="_blank">Co
 
 ## 2. Create a router project directory
 
-As mentioned in [Federation concepts](#federation-concepts), your federated supergraph is represented by a graph router that routes queries to various subgraphs. For this tutorial, we'll use some Apollo-hosted example services as our subgraphs, and we'll set up the Apollo Router in front of them.
+As mentioned in [Federation concepts](#federation-concepts), your federated supergraph is represented by a graph router that routes queries to various subgraphs. For this tutorial, we'll use some Apollo-hosted example services as our subgraphs, and we'll set up the GraphOS Router in front of them.
 
 <Note>
 
-The Apollo Router is a high-performance, precompiled Rust executable that acts as the router for a supergraph.
+The GraphOS Router is a high-performance, precompiled Rust executable that acts as the router for a supergraph.
```
Comment (shorgi) [inline docs/source/quickstart/setup.mdx:201]:
```suggestion
The GraphOS Router is a high-performance, precompiled Rust executable that is based on the Apollo Router Core and acts as the router for a supergraph.
```

File: docs/source/building-supergraphs/router.mdx [hunk 4]
```diff
@@ -23,26 +23,18 @@ graph LR;
 
 Apollo actively supports the following options for your router:
 
-- **The Apollo Router (recommended)**: This is a high-performance, precompiled Rust binary.
+- **The GraphOS Router (recommended)**: This is a high-performance, precompiled Rust binary based on the Apollo Router Core.
 
     - If you're getting started with federation, we recommend [creating a cloud supergraph](/graphos/quickstart/cloud/) with Apollo GraphOS. With a cloud supergraph, GraphOS provisions and manages your router for you.
-    - You can also host your own Apollo Router instances. [See the Federation Quickstart](../quickstart/setup/) to get started.
+    - You can also host your own Apollo Router Core instances. [See the Federation Quickstart](../quickstart/setup/) to get started.
```
Comment (shorgi) [inline docs/source/building-supergraphs/router.mdx:29]:
```suggestion
    - You can also host your own GraphOS Router instances. [See the Federation Quickstart](../quickstart/setup/) to get started.
```

File: docs/source/federated-types/federated-directives.mdx [hunk 5]
```diff
@@ -439,7 +439,7 @@ To learn more, see the [Incremental migration with `@override`](../entities-adva
 
 <EnterpriseFeature>
 
-This argument is available in Apollo Federation 2.7 and later. It is an [Enterprise feature](/router/enterprise-features) of the Apollo Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). You can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+This argument is available in Apollo Federation 2.7 and later. It is an [Enterprise feature](/router/enterprise-features) of the Apollo Router Core and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). You can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```
Comment (shorgi) [inline docs/source/federated-types/federated-directives.mdx:442]:
```suggestion
This argument is available in Apollo Federation 2.7 and later. It is an [Enterprise feature](/router/enterprise-features) of the GraphOS Router and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/). You can test it out by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
```

File: docs/source/federation-2/moving-to-federation-2.mdx [hunk 6]
```diff
@@ -40,20 +40,20 @@ If you encounter an issue while moving your supergraph to Federation 2, please d
 
 For your Federation 1 supergraph to support Federation 2, you first need to upgrade its gateway to one of the following:
 
-- The Apollo Router. This is a high-performance, precompiled executable that provides many benefits over the Node.js-based gateway.
-    - We recommend moving to the Apollo Router unless you've extended your Node.js gateway with functionality that the Apollo Router doesn't currently support (this is uncommon).
+- The GraphOS Router. This is a high-performance, precompiled executable that provides many benefits over the Node.js-based gateway.
```
Comment (shorgi) [inline docs/source/federation-2/moving-to-federation-2.mdx:43]:
```suggestion
- The GraphOS Router. This is a high-performance, precompiled executable that is based on the Apollo Router Core and provides many benefits over the Node.js-based gateway.
```

File: docs/source/federation-versions.mdx [hunk 7]
```diff
@@ -446,7 +446,7 @@ Subscriptions
 <td>
 
 - Composition now supports defining the `Subscription` type in subgraph schemas.
-- Use of GraphQL subscriptions with a federated graph requires a compatible version of the Apollo Router. [See details.](/router/executing-operations/subscription-support/#prerequisites)
+- Use of GraphQL subscriptions with a federated graph requires a compatible version of the Apollo Router Core. [See details.](/router/executing-operations/subscription-support/#prerequisites)
```
Comment (shorgi) [inline docs/source/federation-versions.mdx:449]:
```suggestion
- Using GraphQL subscriptions with a federated graph requires a compatible version of the Apollo Router Core. [See details.](/router/executing-operations/subscription-support/#prerequisites)
```

**Review by shorgi — APPROVED** [review]

**Comment by Meschreiber** [comment]
Marking as draft until terminology is approved.

**Comment by Meschreiber** [comment]
Closing in favor of https://github.com/apollographql/federation/pull/3072

**Pull request closed without merging**